h/api_client/api_client.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""A Python client for the Hypothesis API."""
+import requests
+import pyramid.traversal
+
+
+class HypothesisAPIClient(object):
```

Comment:
I'm in favour of reducing "stuttering" whereever possible. `from h.api_client import HypothesisAPIClient` stutters to me. I'd rather see `from api_client import Client` or even `from h.api import Client` (although that's not a good idea right now, because the `h.api` namespace is the API application.